README.md

```diff
@@ -6,9 +6,12 @@
 
 # gulp
 **The streaming build system**
-
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Support us][gittip-image]][gittip-url] [![Build Status][travis-image]][travis-url] [![Coveralls Status][coveralls-image]][coveralls-url] [![Gitter chat][gitter-image]][gitter-url]
 
+## What is gulp?
+
+Gulp is a build system, meaning that you can use it to automate common tasks in the development of a website, it's allows you to compile preprocessed CSS, minifying JavaScript and reloading the browser and much more.
```

Comment:
For web development (if that's your thing) it can help you ~~with web development~~ (...)